portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,128 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var map;
+var locationsDict = new Object();
```

Comment:
use const or let instead of var (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)